src/usr/local/www/system_advanced_notifications.php

```diff
@@ -411,6 +476,75 @@
 ))->addClass('btn-info')->setHelp('A test notification will be sent even if the service is '.
 	'not enabled.  The last SAVED values will be used, not necessarily the values displayed here.');
 
+	$form->add($section);
+
+	$section = new Form_Section('Pushover');
+
+	$section->addInput(new Form_Checkbox(
+		'enable_pushover',
+		'Enable Pushover',
+		'Enable Pushover Notifications',
+		$pconfig['enable_pushover']
+		))->setHelp('Check this option to enable Pushover notifications. <br>You will need to create an API key and provide your user key. <a href="https://pushover.net/api#registration" target="_blank">API documentation.</a>');
+
+	$section->addInput(new Form_Input(
+		'pushoverapikey',
+		'API Key',
+		'text',
+		$pconfig['pushoverapikey'],
+		['placeholder' => 'azGDORePK8gMaC0QOYAMyEEuzJnyUi']
+	))->setHelp('Enter your API key required to authenticate with the Pushover API server. <a href="https://pushover.net/apps/build" target="_blank">Create API key here.</a>');
```

Comment:
Reword to avoid addressing the user ("you", "your", etc): https://docs.netgate.com/pfsense/en/latest/references/style-guide.html#what-to-avoid